Software/Core/src/main/java/org/foi/emp/core/DAO/KolegijDAO.java

```diff
@@ -0,0 +1,38 @@
+package org.foi.emp.core.DAO;
+
+import androidx.lifecycle.LiveData;
+import androidx.room.Delete;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+import androidx.room.Update;
+
+import org.foi.emp.core.Entities.Kolegij;
+
+import java.util.List;
+
+public interface KolegijDAO {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    public long[] unosKolegija(Kolegij... kolegiji);
+
+    @Update
+    public void azuriranjeKolegija(Kolegij... kolegiji);
+
+    @Delete
+    public void brisanjeKolegija(Kolegij... kolegiji);
+
+    @Query("SELECT * FROM kolegij")
```

Comment:
Are you sure this is going to work?
Check your model class, tableName is "Kolegij" and you are using "kolegij"

`@Query("SELECT * FROM kolegij")`  -> `@Query("SELECT * FROM Kolegij")`